File: utbot-spring-analyzer/src/main/java/analyzer/TestApplicationConfiguration.java
package analyzer;

import org.springframework.beans.factory.config.BeanFactoryPostProcessor;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class TestApplicationConfiguration {

    @Bean
    public static BeanFactoryPostProcessor utBotBeanFactoryPostProcessor() {
        return new UtBotBeanFactoryPostProcessor();
    }
}

File: utbot-spring-analyzer/src/main/kotlin/config/TestApplicationConfiguration.kt
package config

import org.springframework.beans.factory.config.BeanFactoryPostProcessor
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import post_processors.UtBotBeanFactoryPostProcessor

@Configuration
open class TestApplicationConfiguration {

    @Bean
    open fun utBotBeanFactoryPostProcessor(): BeanFactoryPostProcessor {
        return UtBotBeanFactoryPostProcessor()
    }
}